pcsx2/GameDatabase.cpp

```diff
@@ -14,63 +14,193 @@
  */
 
 #include "PrecompiledHeader.h"
+
 #include "GameDatabase.h"
 
-BaseGameDatabaseImpl::BaseGameDatabaseImpl()
-	: gHash( 9900 )
-	, m_baseKey( L"Serial" )
+#include "fmt/core.h"
+#include "yaml-cpp/yaml.h"
+#include <fstream>
+
+std::string GameDatabaseSchema::GameEntry::memcardFiltersAsString()
 {
+	if (memcardFilters.empty())
+		return "";
+
+	std::string filters;
+	for (u32 i = 0; i < memcardFilters.size(); i++)
```

Comment:
This is trivially changeable to a foreach loop.

Reply:
I could, but I need to track the index so I can skip the delimiter on the final filter anyway.  So it ends up being more or less the same I believe?

Reply:
True, I missed that.

Reply:
If only C++ had a proper `join` / `interleave` 👎